cmd/cluster/context.go

```diff
@@ -201,46 +201,40 @@ func (o *contextOptions) run() error {
 
 func (o *contextOptions) printLongOutput(data *contextData) {
 
+	clusterHeader := fmt.Sprintf("%s -- %s", data.ClusterName, data.ClusterID)
+	fmt.Println(clusterHeader)
+	fmt.Println(strings.Repeat("=", len(clusterHeader)))
+
 	printClusterInfo(o.clusterID)
 
 	printSupportStatus(data.LimitedSupportReasons)
-
-	// Print the Servicelogs for this cluster
+	fmt.Println()
+	printJIRASupportExceptions(data.SupportExceptions)
+	fmt.Println()
 	printServiceLogs(data.ServiceLogs, o.verbose, o.days)
-
-	printJIRAOHSS(data.JiraIssues)
-
+	fmt.Println()
+	printJiraIssues(data.JiraIssues)
+	fmt.Println()
 	printCurrentPDAlerts(data.PdAlerts, data.pdServiceID)
+	fmt.Println()
 
 	if o.full {
-		_ = printHistoricalPDAlertSummary(data.HistoricalAlerts, data.pdServiceID, o.days)
+		printHistoricalPDAlertSummary(data.HistoricalAlerts, data.pdServiceID, o.days)
+		fmt.Println()
 
-		err := printCloudTrailLogs(data.CloudtrailEvents)
-		if err != nil {
-			_, _ = fmt.Fprintf(os.Stderr, "Can't print cloudtrail: %v\n", err)
-			os.Exit(1)
-		}
-	} else {
+		printCloudTrailLogs(data.CloudtrailEvents)
```

Comment:
Is there a reason we're no longer handling the error returned by `printCloudTrailLogs`?

Reply:
This was also a table.Flush() error.  I'll update this as well.

Reply:
See the comments for the related change above.